romanisim/util.py

```diff
@@ -544,3 +544,31 @@ def update_photom_keywords(im, gain=None):
         0 * u.MJy / u.sr)
     im['meta']['photometry']['conversion_microjanskys_uncertainty'] = (
         0 * u.uJy / u.arcsec ** 2)
+
+
+def merge_dicts(a, b):
```

Comment:
Should we update the code to use this in other places?

Reply:
We should, but I haven't done that yet.